src/win32/pugixml.rc

```diff
@@ -0,0 +1,47 @@
+#include <winver.h>
+//#include "../pugixml.hpp"
+// TODO these defines should be moved to ../pugixml.hpp
```

Comment:
Yeah I doubt the resource compiler can ever parse the full header. I think it's fine to leave the version numbers duplicated here, they are already spread across many many files and my usual process is to search for all occurrences of 1.9 and change them to 1.10, so this works :)

Reply:
![20190918_pugixml_windows_version_number](https://user-images.githubusercontent.com/309430/65218173-38998900-dab6-11e9-966d-212542c6e4e1.PNG)

Reply:
We could add 
`# define PUGIXML_VERSION_MAJOR 1`
`# define PUGIXML_VERSION_MINOR 10`
`# define PUGIXML_VERSION_PATCH 0`
`# define PUGIXML_VERSION_NUMBER "1.10.0\0"`
to the `PUGIXML_VERSION` section and
`#ifndef RC_INVOKED` in the `pugixml.hpp` header for the rest of the header.

Reply:
Ah I guess. I think it's fine like this.